_drgn.pyi

```diff
@@ -1499,6 +1499,14 @@ class Module:
     module, it is set to the file's build ID if it is not already set. It can
     also be set manually.
     """
+    object: Object
+    """
+    The object this module was created from.
+
+    For Linux kernel loadable modules, this is a reference to the ``struct
+    module`` this was created from. For other kinds, this is currently an absent
```

Comment:
A pointer (i.e. `struct module *`) feels slightly more natural to me (following the precedent of `drgn.Thread.object`). I'd be fine restricting `drgn.Program.linux_kernel_loadable_module()` to only accept `struct module *` and not `struct module` to make that simpler.

Reply:
Yeah, I was thinking about that as well. I get that value pointers vs reference objects are always confusing to users, so I agree it would probably be better to stick with the convention that's already established.

I'd actually love to try to keep `linux_kernel_loadable_module()` accepting `struct module`. If we can't create a pointer to the object, then we can just leave the `Module.object` field absent as before. If you're ok with that.

The reason being that I really enjoyed the possibility of somebody constructing a `struct module` value Object, and then passing it in. I don't actually know how feasible that would be, but it's kinda fun :)